core/src/main/java/bisq/core/dao/state/blockchain/Tx.java

```diff
@@ -138,21 +132,25 @@ public TxOutput getLastTxOutput() {
 
 
     /**
-     * The locktime is stored in the LOCKUP txOutput, which is the first txOutput.
-     *
-     * @return
+     * The lockTime is stored in the LOCKUP txOutput, which is the first txOutput.
      */
     public int getLockTime() {
         return txOutputs.get(0).getLockTime();
     }
 
+    /**
+     * The unlockBlockHeight is stored in the LOCKUP txOutput, which is the first txOutput.
```

Comment:
I think that should be `... stored in the UNLOCK txOutput ...`

Reply:
Hm. I mean the output of the lockup tx (which will be used for unlock later).

Reply:
Changed to "is stored in the first output of the LOCKUP tx" to make it more clear. I just tested with a breakpoint and its the Lockup tx.

Reply:
I would've thought it more natural to store the unlockBlockHeight in the UNLOCK txOutput since it's only then that the unlockBlockHeight is known. It's also when spending the UNLOCK txOutput that the check for current blockheght vs unlockBlockHeight is relevant.

Reply:
Ah ok. I will re-visit it...

Reply:
Ah yes you are right. The locktime is in the LOCKUp tx and the unlockBlockHeight in the UNLOCK tx. I added adjusted comments to all those fields. Will be in the next PR.